### PR TITLE
verification fix for chain subscription flag

### PR DESCRIPTION
**Pull request by stephanwampouille**

### CODE REVIEW

File: training/introduction/orientation/e573a081-0fc3-49b3-b039-cd72e3772a62.yml [hunk 1]
```diff
@@ -30,5 +30,8 @@ blocks:
     real-world cyberattacks, can be safely used to test your internal defenses. You can access these chains from
     within Operator or over our API.
 answer:
-  script: $('#answer').val()
-  value: 10
+  script: 
+    new Promise((resolve, reject) => {
+      Requests.hq.getChains()
+      .then(chains => resolve(Number(chains[1].ttps.length === $('#answer').val())))})
```
Comment (privateducky) [inline training/introduction/orientation/e573a081-0fc3-49b3-b039-cd72e3772a62.yml:36]: won't this return the second most recent chain?

Comment (stephanwampouille) [reply training/introduction/orientation/e573a081-0fc3-49b3-b039-cd72e3772a62.yml:36]: the [0] is ransom note and I think it always is

Comment (privateducky) [reply training/introduction/orientation/e573a081-0fc3-49b3-b039-cd72e3772a62.yml:36]: bugs bugs bugs. hq.getChains() should always return chains in order of release date (those w/out a release date should be at the bottom)

**Comment by stephanwampouille** [comment]
@privateducky ready for review

**Review by privateducky — APPROVED** [review]

**Pull request merged**